compiler/daml-lf-verify/src/DA/Daml/LF/Verify/Generate.hs

```diff
@@ -267,44 +311,31 @@ genForTyApp updFlag expr typ = do
 -- | Analyse a let binding expression.
 genForLet :: (GenPhase ph, MonadEnv m ph)
   => Bool
```

Comment:
Given how many functions take this argument, maybe worth turning it into a `newtype` or a custom ADT with some docs why and when we turn this off.